07_Day_Functions/07_day_functions.md

```diff
@@ -701,7 +701,7 @@ It Will be covered in other section.
     [(1, 4, 5, 7, 9, 8, 0)]
     ```
 
-1. Write a function called reverseCountries, it takes countries array and first it copy the array and returns the reverse of the original array
+1. Write a function called _reverseCountries_, it takes countries array and first it copy the array and returns the reverse of the original array
```

Comment:
Good refactoring.
Nit: Is there a reason for using `_reverseCountries_` instead of `reverseCountries` ?

Reply:
I used _italic_ instead of `code` since I'm just following the formatting of the exercises here in Day 7. I do find it odd how when it comes to colors, it uses **_bold and italic_**. I'll revisit the formatting of the exercises one day.